projects/packages/search/src/customberg/components/app-wrapper/styles.scss

```
@@ -16,6 +16,9 @@
 		position: absolute;
 		padding-top: 2em;
 
+		// Adjusts z-index for customberg only, to allow colorpicker to display correctly.
+		z-index:90;
```

```suggestion
		z-index: 90;
```

Tiny nit: there's usually a space between the colon and the value. 🙂

How did we arrive at the specific value of `90`? Is it the lowest that would override the overlay?

---

>How did we arrive at the specific value of 90? Is it the lowest that would override the overlay?

That's a good question - and yes, it is the lowest value that would override the overlay. This is because the `.interface-interface-skeleton__sidebar` div has a z-index of 90, so without modifying that as well, this one needs to be at least the same, or higher, than that value.

---

Excellent 👍 I think it's worth nothing that in a comment for our future reference.

`// Needs to have equal z-index to .interface-interface-skeleton__sidebar`